aspnetcore/blazor/javascript-interoperability/import-export-interop.md

```diff
@@ -22,15 +24,17 @@ Blazor provides its own JS interop mechanism based on the <xref:Microsoft.JSInte
 This article describes an alternative JS interop approach specific to client-side components executed on WebAssembly. These approaches are appropriate when you only expect to run on client-side WebAssembly. Library authors can use these approaches to optimize JS interop by checking during code execution if the app is running on WebAssembly in a browser (<xref:System.OperatingSystem.IsBrowser%2A?displayProperty=nameWithType>). The approaches described in this article should be used to replace the obsolete unmarshalled JS interop API when migrating to .NET 7 or later.
 
 > [!NOTE]
-> This article focuses on JS interop in client-side components. For guidance on calling .NET in JavaScript apps, see <xref:client-side/dotnet-interop>.
+> This article focuses on JS interop in client-side components. For guidance on calling .NET in JavaScript apps, see <xref:client-side/dotnet-interop/wasm-browser-app>.
 
 ## Obsolete JavaScript interop API
 
 Unmarshalled JS interop using <xref:Microsoft.JSInterop.IJSUnmarshalledRuntime> API is obsolete in ASP.NET Core in .NET 7 or later. Follow the guidance in this article to replace the obsolete API.
 
 ## Prerequisites
 
-[Download and install .NET 7 or later](https://dotnet.microsoft.com/download/dotnet) if it isn't already installed on the system or if the system doesn't have the latest version installed.
+[Visual Studio](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the **ASP.NET and web development** workload.
+
+When installing Visual Studio, select the **.NET WebAssembly build tools** option from the list of optional components.
```

Comment:
I placed it because the prereqs of the .NET JS article call it out ...

> Install the latest version of the [.NET SDK](https://dotnet.microsoft.com/download/dotnet/).
>
> Install the `wasm-tools` workload in an administrative command shell, which brings in the related MSBuild targets:
>
> ```dotnetcli
> dotnet workload install wasm-tools
> ```
>
> The tools can also be installed via Visual Studio's installer under the **ASP.NET and web development** workload in the Visual Studio installer. Select the **.NET WebAssembly build tools** option from the list of optional components.

I switched this article over to VS because I thought that would be the most common development approach for Blazor apps here, but I can add a tab control to hold both the VS and CLI guidance.

Reply:
@pavelsavara ... When using VS (for the Blazor Import-Export article), is the dev required to install the **.NET WebAssembly build tools**? ... because we say in the non-Blazor (client-side article) that it is required to '[bring] in the related MSBuild targets.'

Reply:
For non-Blazor scenario, wasm build tools were required in .NET 7 (because that was how we shipped the SDK build non-Blazor wasm app). In .NET 8 the Wasm SDK is present-by-default in .NET SDK and thus wasm build tools are not a requirement for non-Blazor as well. Anyway without `wasm-experimental` workload (I'm not sure if that is visible in VS installation) users won't have the `wasmbrowser` to create the app